bot/minibot.py

```diff
@@ -73,6 +80,7 @@ def irc_handler(irc):
         try:
             buf = irc.recv(1024).decode("UTF-8").split("\n")
             for i in buf:
+                print(i)
```

Comment:
Yes for both. MSO asked for this in order to debug it more easily when he remotes in.